### PR TITLE
Integrate codecov.io

### PR DESCRIPTION
**Pull request by GBuella**
Ref: #7

---
This change is [Reviewable](https://reviewable.io/reviews/pmem/syscall_intercept/11)

### CODE REVIEW

File: utils/docker/run-coverage.sh [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+#
+# run-coverage.sh - is called inside a Docker container;
+#                   builds syscall_intercept with the aim of collecting
+#                   information on code coverage
+
+cd $WORKDIR
+if [ -n "$COMPILER" ]; then
+	export CC=$COMPILER
+fi
```
Comment (marcinslusarz) [inline utils/docker/run-coverage.sh]: Not needed.

File: utils/docker/run-coverage.sh [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+
```
Comment (marcinslusarz) [inline utils/docker/run-coverage.sh]: Too many empty lines.

**Comment by sarahjelinek** [comment]
:lgtm:

---

Review status: 0 of 10 files reviewed at latest revision, 2 unresolved discussions, some commit checks failed.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/pmem/syscall_intercept/11#-:-Kl_U5Zm4smVFVKtEqTW:bnfp4nl)*

**Comment by krzycz** [comment]
Reviewed 6 of 10 files at r1, 4 of 4 files at r2.
Review status: all files reviewed at latest revision, 2 unresolved discussions.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/pmem/syscall_intercept/11)*

**Comment by krzycz** [comment]
:lgtm:

---

Review status: all files reviewed at latest revision, 2 unresolved discussions.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/pmem/syscall_intercept/11#-:-KlcgOsnlP7axjlVsyVt:bnfp4nl)*

**Comment by sarahjelinek** [comment]
Reviewed 6 of 10 files at r1, 4 of 4 files at r2.
Review status: all files reviewed at latest revision, 2 unresolved discussions, some commit checks failed.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/pmem/syscall_intercept/11)*

**Comment by sarahjelinek** [comment]
Closing because of merge issues. I did the merge and submitted a new pull request since I don't have permissions to GBuella's repo.

**Pull request closed without merging**